src/MoBi.UI/Views/EditOutputSchemaView.cs

```diff
@@ -33,7 +33,7 @@ public EditOutputSchemaView()
 
       public override void InitializeBinding()
       {
-         _gridViewBinder = new GridViewBinder<OutputIntervalDTO>(gridViewIntervals) { BindingMode = BindingMode.OneWay, ValidationMode = ValidationMode.LeavingRow };
+         _gridViewBinder = new GridViewBinder<OutputIntervalDTO>(gridViewIntervals) { BindingMode = BindingMode.OneWay };
```

Comment:
It's a bit less convenient without `LeavingRow`, but with it, you can create the invalid state, and we should prevent the parameter from being changed to create an invalid output interval.

Reply:
Meaning that we will need to edit start/end time always in the "correct" order.
E.g. if I add a second time interval and would like to change it from [0h .. 24h] to e.g. [25h .. 50h] - I would need to change the **end time** first, and then a start time. Little bit annoying, but I can live with it.

<img width="188" height="105" alt="Image" src="https://github.com/user-attachments/assets/e95760a9-c380-4025-aaee-89569900f27d" />